config/crd/kustomization.yaml

```diff
@@ -18,8 +18,8 @@ patches:
 
 # [CERTMANAGER] To enable cert-manager, uncomment all the sections with [CERTMANAGER] prefix.
 # patches here are for enabling the CA injection for each CRD
-#- path: patches/cainjection_in_modules.yaml
-#- path: patches/cainjection_in_managedclustermodules.yaml
+- path: patches/cainjection_in_modules.yaml
+#- path: patches/cainjection_in_preflightvalidations.yaml
```

Comment:
why do we need it if it is commented out?

Reply:
In case we ever want to enable it. When creating CRDs, `operator-sdk` adds those commented out lines. I fixed them and added the missing ones for consistency.